### PR TITLE
Add how to publish new version document.

### PR DESCRIPTION
**Pull request by rantan**

### CODE REVIEW

File: doc/release-process.md [hunk 1]
```diff
@@ -1,29 +1,99 @@
...
+===============
+
+See also [Tapyrus Core Release Process](https://github.com/chaintope/tapyrus-core/blob/master/doc/release-process.md). In general, Tapyrus Signer and Tapyrus Core will be released at the same 
+time.
+
+Before every minor and major release:
+
+* Update version in `Cargo.toml`. See also [vertioning rule](https://github.com/chaintope/tapyrus-core/blob/master/doc/tapyrus/versioning_rule.md).
+* Write release notes (see below)
+* Update Testnet environment. Tapyrus Testnet environment documents are [here](https://drive.google.com/drive/folders/1-J6GE77dH2u5giKoNsuQr24v02bQJjZp) (This is internal document in Chaintope Inc.)
```
Comment (azuchi) [inline doc/release-process.md]: I think, link to Google drive does not necessary.

File: doc/release-process.md [hunk 2]
```diff
@@ -1,29 +1,99 @@
...
+
+Write release notes. git shortlog helps a lot, for example:
+
+    git shortlog --no-merges v(current version, e.g. 0.2.0)..v(new version, e.g. 0.3.0)
+
+Generate list of authors:
+
+    git log --format='- %aN' v(current version, e.g. 0.2.0)..v(new version, e.g. 0.3.1) | sort -fiu
+
+Create release note in `doc/release-notes/tapyrus/`.
```
Comment (azuchi) [inline doc/release-process.md]: `doc/release-notes/` is correct.

File: doc/release-process.md (hunk 1 above)
Comment (rantan) [reply doc/release-process.md]: fixed!

File: doc/release-process.md (hunk 2 above)
Comment (rantan) [reply doc/release-process.md]: fixed!

**Pull request merged**